mobile/src/main/java/org/openhab/habdroid/core/connection/Connection.java

```diff
@@ -54,5 +54,5 @@
     /**
      * @return Whether the this connection is currently reachable.
      */
-    boolean checkReachabilityInBackground();
+    boolean checkReachability();
```

Comment:
If you're changing this method name, please add a statement to the Javadoc saying this must not be called from the main thread.